src/main/java/no/nav/tag/tiltaksgjennomforingprosess/domene/journalpost/Journalpost.java

```diff
@@ -6,20 +6,27 @@
 @Data
 public class Journalpost {
     private final static String JOURNALPOST_TYPE = "INNGAAENDE";
-    private final static String BEHANDLINGSTEMA = "ab0422";
     private final static String KANAL = "NAV_NO";
     private final static String TEMA = "TIL";
-
     final static String TITTEL = "Avtale om arbeidstrening";
-    public final static String EKSTREF_PREFIKS = "AVT";
 
     private final String journalposttype = JOURNALPOST_TYPE;
-    private final String behandlingsTema = BEHANDLINGSTEMA;
     private final String kanal = KANAL;
     private final String tema = TEMA;
-    private String eksternReferanseId;
+    private final String tittel = TITTEL;
 
     private Bruker bruker;
-    private final String tittel = TITTEL;
+    private String eksternReferanseId;
+    private String behandlingsTema;
+    private Boolean tilArena;
     private List<Dokument> dokumenter;
+
+
+    public void setTilArena(boolean tilArena) {
```

Comment:
Denne og neste bør fjernes. Vi har allerede getter og setter for tilArena, og det er best å bruke med de vanlige navnene.